backend/main.go

```diff
@@ -28,11 +42,24 @@ func main() {
 		}
 	}(server)
 
-	server.Port = 8080
-	server.Host = "0.0.0.0"
+	server.Port = cfg.Port
+	server.Host = cfg.Host
 
 	if err := server.Serve(); err != nil {
 		log.Panic().Err(err).Msg("server error")
 	}
 
 }
+
+func initLog(cfg *Config) {
+	var err error
+
+	host, err = os.Hostname()
+	if err != nil {
+		log.Panic().Err(err).Msg("unable to get hostname")
+	}
+
+	zerolog.SetGlobalLevel(zerolog.DebugLevel)
```

Comment:
Let's move all of this to the main function. No need to have a separate function call for these changes.